blockchain/genesis/genesis.go

```diff
@@ -361,3 +372,18 @@ func (r *Rewarding) FoundationBonus() *big.Int {
 	}
 	return val
 }
+
+// InitAdminAddr returns the init admin address
+func (c *Config) InitAdminAddr() address.Address {
+	addrStr := c.InitAdminAddrStr
+	// For backward compatible, if missing, use https://github.com/iotexproject/iotex-bootstrap/blob/master/genesis_mainnet.yaml#L3
+	// as the config protocol admin.
+	if addrStr == "" {
+		addrStr = "io1uqhmnttmv0pg8prugxxn7d8ex9angrvfjfthxa"
```

Comment:
@raullenchai be aware that this will be the default address who can mutate the on-chain config